generators/languages/generator.ts

```diff
@@ -188,6 +187,21 @@ export default class LanguagesGenerator extends BaseApplicationGenerator<
     return this.delegateTasksToBlueprint(() => this.configuring);
   }
 
+  get composing() {
+    return this.asComposingTaskGroup({
+      async bootstrap() {
+        // Make sure generators languages callbacks are correctly initialized.
+        if ((this.jhipsterConfigWithDefaults as ClientConfig).clientFramework === 'angular') {
+          await this.dependsOnBootstrap('angular');
```

Comment:
Got it now, thank you so much for the sample (will pin the PR). A bit worried about it IMHO that the client generator need to know the implementors: Don't you think an extension-point-like solution would be to think about (like in the [good old OSGI](https://vogella.com/blog/getting-started-with-osgi-declarative-services/#1-introduction) days? 
1. Source would declare the API
2. Any implementation answering to this API would register to a whiteboard (static implementation registry)
3. Any call, by any consumer (api call) would lead to a call to each whiteboard-registered Impl.

Would bring extensibility to even unknown component and more loose coupling between generators